src/main.rs

```diff
@@ -45,17 +46,79 @@ amethyst_inspector::inspector![
     HiddenPropagate,
 ];
 
-struct ExampleState;
+struct ExampleState {
```

Comment:
Maybe we should name this `PlayState` or something along those lines?

Reply:
Definitely. It should also probably be moved outside of the `main.rs` file and into its own. I created a `states` folder to store the `PausedState`. Maybe we could move it there ?

Reply:
I think moving it there would be great